Gems/PhysX/Code/Editor/DocumentationLinkWidget.cpp

```diff
@@ -21,8 +21,9 @@ namespace PhysX
             setTextInteractionFlags(Qt::TextBrowserInteraction);
             setOpenExternalLinks(true);
             setAlignment(Qt::AlignCenter);
-            setContentsMargins(60, 15, 60, 15);
+            setContentsMargins(60, 7, 60, 7);
             setWordWrap(true);
+            setStyleSheet(QString::fromUtf8("background-color: rgb(51, 51, 51);"));
```

Comment:
It feels like there ought to be a way to do this with a variable - what happens if we ever want to change the colour scheme, or add different colour scheme options? Is this what the colour settings in the rest of the engine look like?

Reply:
i was looking for something like that, but it appears this is how it was done. if you know of a common colour variable i'm all ears!

Reply:
I can't find it anywhere, so I guess just have to go with this

Reply:
i made this feature request https://github.com/o3de/o3de/issues/2260 to improve this in the future